src/Pages/Login/Login.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { SEND_TO_SERVER } from '../../config';
+
+export default function Login() {
+  const { Kakao } = window;
+  let history = useHistory();
+
+  const KakaoLoginClickHandler = () => {
+    Kakao.Auth.login({
+      success: function (authObj) {
+        sendTokenToServer();
```

Comment:
```suggestion
      success: function (authObj) {
        sendTokenToServer();
```
로그인 성공시 받은 인자로는 아무 것도 하지 않나요?

Reply:
삭제하겠습니다!